ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-evpn.md

```diff
@@ -139,15 +139,23 @@ interface Management1
 | ----- | ------ | --------- |
 | Enabled | 10 |  1 |
 
+#### EVPN DCI Gateway Summary
+
+| Settings | Value |
+| -------- | ----- |
+| Remote Domain Peer Groups | EVPN-OVERLAY-PEERS |
+| L3 Gateway Configured | True |
+| L3 Gateway Inter-domain | True |
+
 #### Router BGP EVPN MAC-VRFs
 
 ##### VLAN aware bundles
 
 | VLAN Aware Bundle | Route-Distinguisher | Both Route-Target | Import Route Target | Export Route-Target | Redistribute | VLANs |
 | ----------------- | ------------------- | ----------------- | ------------------- | ------------------- | ------------ | ----- |
 | B-ELAN-201 | 192.168.255.3:20201 | 20201:20201 | - | - | learned | 201 |
-| TENANT_A_PROJECT01 | 192.168.255.3:11 | 11:11 | - | - | learned | 110 |
-| TENANT_A_PROJECT02 | 192.168.255.3:12 | 12:12 | - | - | learned | 112 |
+| TENANT_A_PROJECT01 | 192.168.255.3:11 | 11:11<br>remote 2:11 | - | - | learned | 110 |
```

Comment:
Is the new line expected, I guess it's just multiple RTs

Reply:
Yes it's expected, that's how we put multiple RTs in the column, separated by <br>. It confused me first time i saw it as well.